src/details/actions.js

```diff
@@ -63,7 +73,7 @@ export const getUsageDetails = (periodEnd, skip) => {
   return dispatch => {
     dispatch({ type: d.LOADING })
 
-    const past30Days = moment(periodEnd).subtract(1, 'months').format()
+    const past30Days = moment(periodEnd).startOf('day').subtract(1, 'months').format()
 
     // Return the mock data
     // dispatch(computeTrends(json))
```

Comment:
Should this be removed as it is commented out?

Reply:
it is also a duplicate from line `93` and `102` which seem to be run no matter what.

Recommend deletion.

Reply:
Yup, this ties in to the import above, will get rid of it for now